### ISSUE TITLE
Funny warning

### ISSUE THREAD

**Issue by polluks** [jsummers/deark#18]
vbcc says
````
vc  -Isrc -c -o obj/modules/cfb.o modules/cfb.c
>       // SecID of first (extra??)
warning 293 in line 350 of "modules/cfb.c": trigraph ??) encountered
>       // Number of (extra??)
warning 293 in line 356 of "modules/cfb.c": trigraph ??) encountered
````
not so funny
````
vc  -Isrc -c -o obj/src/fmtutil-miniz.o src/fmtutil-miniz.c
>static void *def_alloc_func(void *opaque, size_t items, size_t size) { (void)op
warning 153 in line 1062 of "/foreign/miniz.h": statement has no effect
        included from file "src/fmtutil-miniz.c":15
>static void def_free_func(void *opaque, void *address) { (void)opaque, (void)ad
warning 153 in line 1063 of "/foreign/miniz.h": statement has no effect
        included from file "src/fmtutil-miniz.c":15
>  (void)pStream;
warning 153 in line 1180 of "/foreign/miniz.h": statement has no effect
        included from file "src/fmtutil-miniz.c":15
````

**Comment by jsummers** [comment]
a) I'm a little surprised that a compiler would warn about trigraphs in comments. I'll fix it, but it could happen again.

b) I'll probably "fix" this, but I'm surprised that a compiler would warn about an idiom that is (or at least used to be) commonly used to _suppress_ warnings.

Note - I'm aware that the miniz code has a number of other issues that static analyzers might complain about.

**Comment by jsummers** [comment]
Addressed by commit 653d2120fc7c and commit ce3bf77a0a54.
